testutil/testclient/testqueryclients/servicequerier.go

```diff
@@ -0,0 +1,61 @@
+package testqueryclients
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+
+	"github.com/pokt-network/poktroll/testutil/mockclient"
+	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
+)
+
+// services is a map of: serviceId -> Service
+var services map[string]sharedtypes.Service
+
+func init() {
+	services = make(map[string]sharedtypes.Service)
+}
```

Comment:
I think this can be simplified:

```suggestion
var services = make(map[string]sharedtypes.Service)
```